packages/xo-server-sdn-controller/src/index.js

```diff
@@ -406,6 +408,7 @@ class SDNController extends EventEmitter {
     xoPif,
     vni,
     encrypted = false,
+    mtu = 0,
```

Comment:
IMHO you should migrate these default value in the exposed public API, because you appear to always use it explicitly when you call the private methods.